src/test/java/itm/oss/splitter/SettlementTest.java

```diff
@@ -0,0 +1,41 @@
+package itm.oss.splitter;
+
+import org.junit.jupiter.api.Test;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * JUnit tests for Settlement.suggest()
+ */
+public class SettlementTest {
+
+    @Test
+    void test_singlePayment_whenOnePersonOwesAnother() {
+        // One debtor (-30) and one creditor (+30)
+        Balance b = new Balance();
+        b.put("Alice", new BigDecimal("-30"));
+        b.put("Bob", new BigDecimal("30"));
+
+        ArrayList<Payment> result = Settlement.suggest(b);
+
+        // Expect one payment: Alice → Bob: 30
+        assertEquals(1, result.size());
+        assertEquals("Alice", result.get(0).getFrom());
+        assertEquals("Bob", result.get(0).getTo());
+        assertEquals(0, result.get(0).getAmount().compareTo(new BigDecimal("30")));
+    }
+
+    @Test
+    void tinyAmountTest() {
```

Comment:
Thanks for your great work! I have a small suggestion on your code. The current name tinyAmountTest is a bit vague what is the purpose of this function.

I suggest renaming it to:
```suggestion
    void test_ignoresPayments_whenBalanceIsBelowThreshold() {
```